### PR TITLE
GA conversationMember types for Teams chats

### PR DESCRIPTION
**Pull request by jessiechan27**

### CODE REVIEW

**Review by HACKED-need-HELP — APPROVED** [review]

**Review by abshar-teams — APPROVED** [review]

File: api-reference/v1.0/resources/microsoftaccountuserconversationmember.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Inherits from [conversationMember](../resources/conversationmember.md).
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|displayName|String|Display name of the user. Inherited from [conversationMember](../resources/conversationmember.md).|
+|id|String|Membership ID that represents this resource. Inherited from [entity](../resources/entity.md).|
+|roles|String collection|Special roles for this user. Inherited from [conversationMember](../resources/conversationmember.md).|
+|userId|String|ID of the user.|
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/microsoftaccountuserconversationmember.md:25]: What is the difference between the ID inherited from `entity` and this userID? Is there more information we can provide to developers to use this property?

File: api-reference/v1.0/resources/skypeforbusinessuserconversationmember.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Inherits from [conversationMember](../resources/conversationmember.md).
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|displayName|String|Display name of the user. Inherited from [conversationMember](../resources/conversationmember.md).|
+|id|String|Membership ID that represents this resource. Inherited from [entity](../resources/entity.md).|
+|roles|String collection|Special roles for this user. Inherited from [conversationMember](../resources/conversationmember.md).|
+|tenantId|String|ID of the tenant that the user belongs to.|
+|userId|String|ID of the user.|
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/skypeforbusinessuserconversationmember.md:27]: Similar than above: What is the difference between the ID inherited from entity and this userID? Is there more information we can provide to developers to use this property?

**Review by isvargasmsft — APPROVED** [review]

File: api-reference/v1.0/resources/microsoftaccountuserconversationmember.md (hunk 1 above)
Comment (jessiechan27) [reply api-reference/v1.0/resources/microsoftaccountuserconversationmember.md:25]: The ID inherited from entity is the id generated in a specific chat whenever a new chat member that joins. It's unrelated to the user type. userID is the AAD user ID for chat members who are AAD users

File: api-reference/v1.0/resources/skypeforbusinessuserconversationmember.md (hunk 2 above)
Comment (jessiechan27) [reply api-reference/v1.0/resources/skypeforbusinessuserconversationmember.md:27]: The ID inherited from entity is the id generated in a specific chat whenever a new chat member that joins. It's unrelated to the user type. userID is the AAD user ID for chat members who are AAD users

File: api-reference/v1.0/resources/microsoftaccountuserconversationmember.md (hunk 1 above)
Comment (jessiechan27) [reply api-reference/v1.0/resources/microsoftaccountuserconversationmember.md:25]: The ID inherited from entity will be an MRI that looks entirely different and the format will be consistent across all chat members in the roster. Developers can tell that the userID is an AAD user ID by seeing that the chat member is of the microsoftAccountUserConversationMember type

File: api-reference/v1.0/resources/skypeforbusinessuserconversationmember.md (hunk 2 above)
Comment (jessiechan27) [reply api-reference/v1.0/resources/skypeforbusinessuserconversationmember.md:27]: The ID inherited from entity will be an MRI that looks entirely different and the format will be consistent across all chat members in the roster. Developers can tell that the userID is an AAD user ID by seeing that the chat member is of the microsoftAccountUserConversationMember type

File: api-reference/v1.0/resources/microsoftaccountuserconversationmember.md (hunk 1 above)
Comment (jessiechan27) [reply api-reference/v1.0/resources/microsoftaccountuserconversationmember.md:25]: These changes exist in beta and may cause issues if the properties names are changed or property values are duplicated

Comment (jessiechan27) [reply api-reference/v1.0/resources/microsoftaccountuserconversationmember.md:25]: I'll update the description to let the developer know that it is the AAD user id

File: api-reference/v1.0/resources/skypeforbusinessuserconversationmember.md (hunk 2 above)
Comment (jessiechan27) [reply api-reference/v1.0/resources/skypeforbusinessuserconversationmember.md]: See above comment replies

**Pull request merged**